cmd/kpod/common.go

```diff
@@ -133,3 +136,19 @@ func validateFlags(c *cli.Context, flags []cli.Flag) error {
 	}
 	return nil
 }
+
+// getCachedCreds returns the DockerAuthConfig of a registry if stored in the authfile
+func getCachedCreds(registry string, sc *types.SystemContext) (*types.DockerAuthConfig, error) {
+	if registry == "" {
+		return nil, nil
+	}
+	user, pass, err := config.GetAuthentication(sc, registry)
```

Comment:
We had talked at one point that we'd first try to get the creds from the default "kpod" location, then if not there, we'd try the default Docker location.  Does that still hold?  I'm not seeing that in this code, but it might be buried under calls.

Reply:
Yes we should fail over to that.

Reply:
yup, that is happening in `GetAuthentication` in `containers/image/pkg/docker/config/config.go`. It checks both `~/.docker/config.json` and the legacy path `~/.dockercfg` if it cannot be found in `XDG_RUNTIME_DIR/containers/auth.json`.

Reply:
👍